### DISCUSSION TITLE
wire model is completely ignore the zero

### DISCUSSION THREAD

**Discussion by ShekhSaifuddin007** [livewire/livewire#2069] [1. Help]
If some input field has a default value like `25.00` it completely ignores that, if it has `25.01` it shows why it ignores those zero's? I need full-floating value in my input field, how to do that?

`$single_profit_percent = 25.00`

### blade

```blade
<input type="text" wire:model.defer="single_profit_percent" class="form-control" placeholder="Exc.tax">
```

**Comment by ousid** [comment]
Hey, @ShekhSaifuddin007 

Try to cast your property to float type:
```php
(float) $single_profit_percent
```

For more info, take a look in [PHP doc](https://www.php.net/language.types.type-juggling)
